### ISSUE TITLE
Is there any way to debug SceShellCore?

### ISSUE THREAD

**Issue by charlyzard** [xemio/ps4debug#7]
Debug watch crashes when attaching to SceShellcore. Is there any way to debug said process?

**Comment by charlyzard** [comment]
I'd like to debug the libSceFsInternalForVsh library used by SceShellcore.

I tried your suggestion of injecting into SceSysCore (the exact proc name is "SceSysCore.elf"). The payload seems to load properly as I get the two notifications "ps4debug" and "golden <3". 

But something seems to be missing as when a breakpoint or watchpoint is hit the process being debugged stops but debug watch doesn't know the process hit that breakpoint.

The usual "interrupt hit (thread: ..." notification isn't shown on the ps4 and the register windows on debug watch is not updated.

But if you press the "resume" button on debug watch, the process continues normally.

So injecting SceSysCore almost works but somehow when an interrupt is hit the client doesn't know it was hit.

I've tried injecting different processes but haven't found one that works. I'll keep searching.
